### ISSUE TITLE
Hovering a variable value disappears too quickly

### ISSUE THREAD

**Issue by capegreg** [microsoft/vscode#224230]
Issue is related to: https://github.com/microsoft/vscode/pull/214186

The short delay makes copying variable values on hover impossible, whereas in prior versions it was working.

Steps to Reproduce:

1. Use latest version of Insiders
2. Place breakpoint on a variable.
3. Hover a variable to show its tool tip value.
4. Double-clicked the hovered value and select it.
5. Press Control-C to copy tool tip value. Hovered value disappears immediately.

Version: 1.92.0-insider (user setup)
Commit: 73c35330d03213d41829d35415cc7cb1df455689
Date: 2024-07-29T14:26:24.414Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by roblourens** [comment]
This is working normally for me. Could you get a screen recording? https://gifcap.dev/ is one option

**Comment by capegreg** [comment]
Hello. The issue is really when copying the hovered value, the hover disappears immediately after moving the I-beam away from the hovered value. In previous versions, the hovered value was visible much longer. Doing a Ctrl-C of the hovered value was always successful.

![hover](https://github.com/user-attachments/assets/b2ec101a-7b87-46f0-bccc-072a22a87c69)

**Comment by roblourens** [comment]
I think that if you move the mouse away from the hover, then I'd expect it to hide, are you sure that's new? But then it looks like ctrl+C caused it to hide once? I can't repro that.

**Comment by capegreg** [comment]
I tried to find out where this behavioral divergence started for me. I installed archived versions of VSCode, beginning with the June 2022 version (1.69) through to the latest version. Although I didn’t measure it precisely, I noticed that the delay before the tooltip hides after moving the pointer away seems to decrease as I moved towards the April 24 version, then almost immediately on subsequent versions. I can select and copy the content from the tooltip with Ctrl+C in every version as long as my pointer remains within its boundary. While the tooltip is expected to hide when the pointer leaves the hovered area, having a little more time before so helps. I accept that it’s possible that the current delay is the intended design change for hover. Thanks.
